File: AR-VR/devAyushDubey_ayushdubey_2024csit1122_2/Patterns/pattern11.cpp
#include <iostream>
using namespace std;

int main(){
    int n,i,j,num=1;
    cin >> n;
    for(i=1;i<=n;i++){
        for(j=0;j<i;j++)
            cout<<num++<<"\t";
        cout<<endl;
    }
    //write your code here
    
}

File: IOT/shakap12_ShashwatKapoor_1923ec1177_3/Patterns/Pattern11.java
import java.util.*;

public class Main {

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);

        // write ur code here
        int c=1;
        int n=scn.nextInt();
        for(int i=0;i<n;i++){
            for(int j=0;j<=i;j++){
                System.out.print(c+"	");
                c++;
            }
            System.out.println();
        }

    }
}